### PR TITLE
Add Gauss-Lobatto quadrature

### PR DESCRIPTION
**Pull request by nicoguaro**
- Add [Gauss-Lobatto quadrature](https://en.wikipedia.org/wiki/Gaussian_quadrature#Gauss.E2.80.93Lobatto_rules)
- Formatted code according to PEP8 (except for hyperlinks that are too long for that)

### CODE REVIEW

File: sympy/integrals/quadrature.py [hunk 1]
```diff
@@ -486,14 +509,89 @@ def gauss_jacobi(n, alpha, beta, n_digits):
...
+            - (2*n+alpha+beta+2) / (n+alpha+beta+S.One) *
+            (gamma(n+alpha+1)*gamma(n+beta+1)) /
+            (gamma(n+alpha+beta+S.One)*gamma(n+2)) *
+            2**(alpha+beta) / (pd.subs(x, r) * pn.subs(x, r))).n(n_digits))
+    return xi, w
+
+
+def gauss_lobatto(n, n_digits):
+    r"""
+    Computes the Gauss-Legendre quadrature [1]_ points and weights.
```
Comment (certik) [inline sympy/integrals/quadrature.py]: Gauss-Legendre -> Gauss-Lobatto?

**Comment by certik** [comment]
I think you should merge this PR with your other PR #10260, so that the new feature and tests are added at the same time.

File: sympy/integrals/quadrature.py (hunk 1 above)
Comment (nicoguaro) [inline sympy/integrals/quadrature.py]: Yes, my bad

**Comment by sylee957** [comment]
I think this PR is already merged in #11728.

**Comment by certik** [comment]
@sylee957 thanks.

**Pull request closed without merging**